cmd/oras/root/manifest/index/update.go

```diff
@@ -64,9 +67,18 @@ Example - Merge manifests from the index 'v2-windows' to the index 'v2':
 
 Example - Update an index and tag the updated index as 'v2.1.0' and 'v2':
   oras manifest index update localhost:5000/hello@sha256:99e4703fbf30916f549cd6bfa9cdbab614b5392fbe64fdee971359a77073cdf9 --add linux-amd64 --tag "v2.1.0" --tag "v2"
+
+Example - Update an index and save it locally to index.json, auto push will be disabled:
```

Comment:
nit: I think "auto push will be disabled" looks a bit too-much. How about something like 
> Update the index and save it locally to index.json without pushing it.

?

Reply:
I think `auto push will be disabled` is clearer and emphasizes that there will be no push when `--output` is used.

Reply:
We can update it later since the current wording is consistent with `index create`.